manifests/charts/cloudcore/Chart.yaml

```diff
@@ -3,5 +3,6 @@ name: cloudcore
 version: 0.1.0
 appVersion: 1.12.0
 description: The KubeEdge cloudcore component.
+kubeVersion: ">= 1.16.0-0 < 1.23.0-0"
```

Comment:
Hard code is not so elegant, and is it necessary to write the k8s version here? @my-git9

Reply:
I think it is necessary to a certain extent, it can prevent uninformed users from deploying kubeedge on unsupported k8s versions